Sources/AirRobeWidget/Service/Endpoints/AirRobeWidgetEndpoints.swift

```diff
@@ -79,7 +79,7 @@ extension AirRobeEndpoint {
             "anonymous_id": UIDevice.current.identifierForVendor?.uuidString ?? "",
             "session_id": sessionId,
             "external_order_id": orderId,
-            "split_test_variant": "default",
+            "split_test_variant": UserDefaults.standard.TargetSplitTestVariant?.splitTestVariant ?? "default",
```

Comment:
Maybe we want to get rid of the keyword 'Target' here.

Reply:
All updated. I'd like to get this merged before I proceed with another widget variant implementation.